Description: do not let flite add a dummy rpath
Author: Samuel Thibault <sthibault@debian.org>
Forwarded: https://github.com/festvox/flite/pull/5

Index: flite/main/Makefile
===================================================================
--- flite.orig/main/Makefile
+++ flite/main/Makefile
@@ -79,7 +79,6 @@ LOCAL_CLEAN = $(BINDIR)/flite$(EXEEXT) $
               flite_voice_list.c
 
 ifdef SHFLAGS
-flite_LIBS_flags += -Wl,-rpath $(LIBDIR) 
 shared_libs: $(SHAREDLIBS)
 else
 shared_libs: nothing
